WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m

```diff
@@ -238,6 +239,37 @@ - (BOOL)isWDHittable
   return nil == result ? NO : result.hittable;
 }
 
+/*! Whether the element is truly hittable based on XCUIElement.hittable */
+- (BOOL)isWDNativeHittable
+{
+  XCUIApplication *app = [XCUIApplication fb_activeApplication];
```

Comment:
this might be suboptimal. Sometimes, the interface method might be called on XCUIElement instance, which would result in element->snapshot->element chain. It is necessary to modify `fb_valueForWDAttributeName` to avoid that.